dat-schema/_Core.gql

```diff
@@ -5710,6 +5831,31 @@ type ItemVisualReplacement {
   _: bool
 }
 
+# Added 3.23
+type LabyrinthCraftOptions {
```

Comment:
already added in other file, please rebase

Reply:
my bad